plugins/ledger/config/initializers/ledger/client.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+#
+# basic ledger event client
+#
+require 'faraday'
```

Comment:
plugin should depend on faraday just in case ... `gem.add_runtime_dependency 'faraday'`